crates/forge/src/compiled_raw.rs

```diff
@@ -1,69 +1,10 @@
-use cairo_felt::Felt252;
 use cairo_lang_sierra::program::Program;
-use conversions::IntoConv;
-use forge_runner::compiled_runnable::{FuzzerConfig, ValidatedForkConfig};
-use forge_runner::expected_result::ExpectedTestResult;
-use num_bigint::BigInt;
 use serde::Deserialize;
-use starknet::core::types::BlockId;
-use starknet::core::types::BlockTag::Latest;
+use snforge_test_collector_abstractions::{CrateLocation, TestCaseRaw};
 
 #[derive(Debug, Clone, Deserialize)]
 pub(crate) struct CompiledTestCrateRaw {
     pub sierra_program: Program,
     pub test_cases: Vec<TestCaseRaw>,
     pub tests_location: CrateLocation,
 }
```

Comment:
This can be moved to the common crate too

Reply:
No it cannot - it depends on `cairo-lang-sierra`, it would always link the same version in both collector and forge.